v2/pkg/common/annotations/annotations.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package annotations
+
+const PerResourceSecretAnnotation = "serviceoperator.azure.com/credential-from"
+
+// ReconcilePolicyAnnotation describes the reconcile policy for the resource in question.
+// A reconcile policy describes what action (if any) the operator is allowed to take when
+// reconciling the resource.
+// If no reconcile policy is specified, the default is "run"
+const ReconcilePolicyAnnotation = "serviceoperator.azure.com/reconcile-policy"
```

Comment:
Suffix mirrors the package name and can be removed.

```suggestion
const ReconcilePolicy = "serviceoperator.azure.com/reconcile-policy"
```

Reply:
Hmm actually, one interesting thing here is key vs value. This is the reconcile policy annotation key, but there's also a reconcile policy annotation value (which is currently in `reconciler` but IMO should be here.

We should adapt a common naming pattern for both of these.

`annotations.ReconcilePolicy` (for the annotation) and `annotations.ReconcilePolicyValue` (the enum type), and `annotations.ReconcilePolicySkip` or `annotations.ReconcilePolicyManage` for the actual values?